.circleci/config.yml

```diff
@@ -902,13 +902,19 @@ jobs:
       - checkout
       - attach_workspace:
           at: build
-      - run: scripts/bytecodecompare/storebytecode.sh && cp -v report.txt bytecode-report-ubuntu.txt
+      - run: mkdir test-cases/
+      - run: cd test-cases && ../scripts/isolate_tests.py ../test/
+      - run: cd test-cases && ../scripts/bytecodecompare/prepare_report.py ../build/solc/solc --interface standard-json && mv -v report.txt ../bytecode-report-ubuntu-json.txt
+      - run: cd test-cases && ../scripts/bytecodecompare/prepare_report.py ../build/solc/solc --interface cli && mv -v report.txt ../bytecode-report-ubuntu-cli.txt
       - store_artifacts:
-          path: report.txt
+          path: report-json.txt
+      - store_artifacts:
+          path: report-cli.txt
```

Comment:
Where do these two files come from?

Reply:
Right. This should have been `bytecode-report-ubuntu-cli.txt`.